install/k8s/contiv/contiv.yaml

```diff
@@ -1,4 +1,99 @@
 ---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
```

Comment:
Contiv works with the default k8s ClusterRoleBindings.  Curious why this is needed? for RBAC?

Reply:
RBAC